lang/cs/Org.Apache.REEF.IMRU.Examples/MapperCount/MapperCount.cs

```diff
@@ -66,8 +72,16 @@ public int Run(int numberofMappers)
                             GenericType<IntSumReduceFunction>.Class)
                         .Build())
                     .SetPartitionedDatasetConfiguration(
-                        RandomInputDataConfiguration.ConfigurationModule.Set(RandomInputDataConfiguration.NumberOfPartitions,
+                        RandomInputDataConfiguration.ConfigurationModule.Set(
+                            RandomInputDataConfiguration.NumberOfPartitions,
                             numberofMappers.ToString()).Build())
+                    .SetResultHandlerConfiguration(
+                        TangFactory.GetTang()
+                            .NewConfigurationBuilder(fileSystemConfig)
+                            .BindImplementation(GenericType<IObserver<int>>.Class,
+                                GenericType<WriteResultHandler<int>>.Class)
+                            .BindNamedParameter(typeof(ResultOutputLocation), outputFile)
```

Comment:
I had a discussion with @markusweimer . Having one output file like in Hadoop is ok and multiple files do not give us anything beyond it.